cfpb/sbl-frontend
# [Bug] Triage existing linting issues

Went through several linter issues and fixed most

## Changes

- Went through and found areas of easy linter improvements and fixed them.
- Went through and found areas referencing this ticket and either fixed them, or put them under the umbrella of the post-mvp linter issues.

## How to test this PR

1. Pull the branch for the sbl-frontend repo
2. Restart the stack as necessary
3. Check the github pipeline and make sure that linters are green
4. Run the e2e tests and verify no new failures
